internal/internal_worker.go

```diff
@@ -1286,20 +1289,12 @@ func (aw *WorkflowReplayer) replayWorkflowHistory(logger log.Logger, service wor
 			for _, d := range completeReq.Commands {
 				if d.GetCommandType() == enumspb.COMMAND_TYPE_CONTINUE_AS_NEW_WORKFLOW_EXECUTION {
 					if last.GetEventType() == enumspb.EVENT_TYPE_WORKFLOW_EXECUTION_CONTINUED_AS_NEW {
-						inputA := d.GetContinueAsNewWorkflowExecutionCommandAttributes().GetInput()
```

Comment:
I removed it based on the conversation we had here https://github.com/temporalio/sdk-go/issues/978 we said

"we need to stop using payload comparison anyways. I don't believe we need to check input/output data"

Reply:
Hrmm, I may have been thinking more activities and child workflows and such inside the workflow replay check instead of the replayer. But this might be ok. Let me bring it up in Slack to get others' opinions.

Reply:
We can try to keep this, but we can't leave it in its current form because the binary comparison is not reliable.

Reply:
Conferred, good to remove!